### PR TITLE
Llm pretrained

### PR DESCRIPTION
**Pull request by penjc**
# Description
Add three pages and add new dependencies.
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] [local build the book](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [ ] review the artifacts from local build
- [ ] [local build the slides](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [ ] Unit test if appliable

# Checklist:

- [ ] My code follows the [style guidelines](https://github.com/ocademy-ai/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have applied tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: open-machine-learning-jupyter-book/_config.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ execute:
     - 'ml-advanced/unsupervised-learning.ipynb'
     - 'data-science/data-science-in-the-cloud/the-azure-ml-sdk-way.ipynb'
     - 'llm/basic/transformer.ipynb'
+    - 'llm/basic/language-modelling.ipynb'
+    - 'llm/pretrained-model/pretraining-on-unlabeled-data.ipynb'
```
Comment (penjc) [inline open-machine-learning-jupyter-book/_config.yml:28]: the codes in the file need to train the model, and execution times out. i ran the outputs straight from the local.

Comment (Lola-jo) [reply open-machine-learning-jupyter-book/_config.yml:28]: You can just leave the llm chapters unloaded, refer to deep learning for details. in that case, remember to upload all the files with the output.

File: open-machine-learning-jupyter-book/llm/basic/language-modelling.ipynb [hunk 2]
```diff
@@ -0,0 +1,1724 @@
+{
```
Comment (Lola-jo) [inline open-machine-learning-jupyter-book/llm/basic/language-modelling.ipynb:1]: <p>Can this model be downloaded locally and uploaded to cos? Try to do so if it is possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (Lola-jo) [inline open-machine-learning-jupyter-book/llm/basic/language-modelling.ipynb:1]: <p>Can this model be downloaded locally and uploaded to cos? Try to do so if it is possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (penjc) [reply open-machine-learning-jupyter-book/llm/basic/language-modelling.ipynb:1]: Models can be downloaded locally and uploaded to cos.
But it is not possible to modify the code to load from cos directly. Here the transformers package is used to load the model, it is not possible to get the model from the cos link, only from local or huggingface.

only one way: download from hub and load from loacl. I don't think that's necessary.
import requests
from transformers import BertModel

model_url = "https://static-1300131294.cos.ap-shanghai.myqcloud.com/images/llm/bert.safetensors"
model_file = "bert_model.pth"
response = requests.get(model_url)
with open(model_file, "wb") as f:
    f.write(response.content)

BERTmodel = BertModel.from_pretrained(model_file)

Comment (Lola-jo) [reply open-machine-learning-jupyter-book/llm/basic/language-modelling.ipynb:1]: OK，good job！

**Pull request merged**